### ISSUE TITLE
原以为技术无国界，敢情现在技术有政治？最新的版本 Stand with Hong Kong 几个 意思？

### ISSUE THREAD

**Issue by maobiwen** [notepad-plus-plus/notepad-plus-plus#8658]
原以为程序交流的世界，能不夹杂任何政治因素在里面，git最后的一片净土，都被你这个notepad++污染
你怎么不stand with美国的那些暴徒？

钓鱼岛、南海九段线内、台湾、香港、澳门，自古就是中国的一部分，过去是，将来是，永远都是，去你妈stand with hongkong 。大家开新的分支去吧。

**Comment by shaiyu** [comment]
别激动,  可能作者家里着火了,  或者全家炸了,  家里事太急,  家里天天出事

**Comment by maobiwen** [comment]
> 别激动, 可能作者家里着火了, 或者全家炸了, 家里事太急, 家里天天出事

也可能是作者老婆出轨了，绿帽子戴得太多。

**Comment by snys98** [comment]
>我原以为技术界可能是最后一个和任何政治狗屎都没有关系乌托邦。 但是现在它已经被你的双标政治正确污染了。
烦请不要再对其他种族，尤其是中国人做出政治判断。 我们在做我们自己的事请，这和你没有任何关系。 脑子灵光一点，如果你不理解某个事情，那么你应该做的就是保持尊重。 你没有叼到可以对所有事情说长道短，尤其是一件14亿人正在做的事情。
钓鱼岛，南中国海的九点线，台湾，香港和澳门自古以来就是中国的一部分。 它曾经是，将会是，并且将永远是。

I thought the world of programing could be the last utopia that has no connection with any political shit. But now it's been polluted by your double-standard political correct.

Stop putting your political judgement on other races, especially Chinese. We are doing our own business, and it's none of your business. Be brilliant, if you don't understand something, all you should do is to keep respect. You are not great enough to judge everything, especially something that is doing by 1.4 billion people.

The Diaoyu Islands, the Nine-dash Line in the South China Sea, Taiwan, Hong Kong, and Macau have been a part of China since ancient times. It used to be, will be, and will always be.

**Comment by Lin-Buo-Ren** [comment]
光復香港，時代革命！

**Comment by KOLANICH** [comment]
>We are doing our own business, and it's none of your business.

Who are "you"? Your try to claim that PRC is yours is completely wrong. It is just a tale for not so smart people - to pretend that the state you live in is "yours". You are just a miserable slave owned by "People" "Republic" of "China" whose every action is spied on and who is afraid to be prosecuted for even seeing a word against actions done by PRC and who is trying to maintain an own "social credit score".

If you deserved better life, you would have been already in other state. But you don't. I don't approve the US, it also has facial recognition , so do UK, EU, Russia, but we don't have such shit as PRC has. It is PRC that shows how miserable people of other states could have been. And this makes this shit proliferate to other states.

We don't have anything against the words "PRC", "China", "Han", "Uighur". We just absolutely disgusted by the things PRC does:
1. absolute totalitarism in internal affairs
2. exporting totalitarism using by dumping backdoored devices, unwilingly subsidized by people living wituin PRC-controlled territory, at our markets
3. making the deals not mutually beneficial in long term in external affairs (and PRC can really push for them, it is a top military power in the world)

In other words, you must change your mindset first. You should judge not "does this harm my masters, I have no other purpose in the life but to serve my masters, what is profitable to my masters is profitable to me", but "does this harm people, not only belonging to my masters, including me personally, but also to other people, all the ordinary people in the world are my allies, what harms people will harm me, what harms me - that will harm people".

**Comment by Da1Liu** [comment]
F**K you OK?

**Comment by Da1Liu** [comment]
> > We are doing our own business, and it's none of your business.
> 
> @snys98 Who are "you"? Your try to claim that PRC is yours is completely wrong. It is just a tale for not so smart people - to pretend that the state you live in is "yours". You are just a miserable slave owned by "People" "Republic" of "China" whose every action is spied on and who is afraid to be prosecuted for even seeing a word against the actions done by PRC and who is trying to maintain an own "social credit score" just to prove his masters that you are more useful for them than other slaves.
> 
> If you deserved better life, you would have been already in an other state, better than PRC in sense of personal freedoms and human rights. But you don't, because it is you that have chosen so. I don't approve some things the US is doing, it also has facial recognition, so do US, UK, EU, Russia (all the latters are powered by the software and hardware by chinese corporations), but we don't see such shit as PRC has, such as "social credit score" systems or internment camps (US has a lot, but, as I have said, this is completely inacceptable) in the states we live in and hope there will be never so. It is PRC that shows how miserable people of other states could have been. And this makes this shit proliferate to other states.
> 
> We don't have anything against the words "PRC", "China", "Han", "Uighur". We have nothing against the people or the language they spoke or their healthcare traditions (as long as they don't force us to acdept them, i.e. it is clearly inacceptable to go to my home and spit onto the floor, but they can spit onto the floor on their personal homes, if it is their personal homes). We are just absolutely disgusted by the things PRC does:
> 
> 1. absolute totalitarism and human rights violation in internal affairs
> 2. exporting totalitarism by dumping backdoored devices, unwilingly subsidized by people living within PRC-controlled territory, to our markets. Some idiots buy these devices because of their low price, and through their devices PRC spies not only on them, but on me, who doesn't have any Xiaomi or Huawei or any other smart-devices made by Chinese companies prescribed by the Party to insert backdoors.
> 3. making the deals not mutually beneficial in long term in external affairs (and PRC can really push other states for them by a threat to acquire what it wants by military means, since PRC is the top military power in the world now, only the US is the second)
> 
> We don't want more people to get under totalitarian rule. And we especially don't want ourselves to get under a totalitarian rule.
> 
> In other words, you must change your mindset first. You should judge not "does this harm my masters, I have no other purpose in the life but to serve my masters, what is profitable to my masters is profitable to me, because it is my masters who have enforced that rule", but "does this harm people, not only belonging to my masters, including me personally, but also to other people, all the ordinary people in the world are my allies, what harms people will harm me, what harms me - that will harm people".

Bull shit man, you think you know  real PRC from those fake news that your government shows you?what a joke?

**Comment by zhly0** [comment]
free your old grandma，why don't you free black people in US，In 2020，The real bad man is people like you，let‘s talk about people died in Iraq，where is so called freedom. The western is a man pretend to be gentleman with blood in your hand.

**Comment by Lin-Buo-Ren** [comment]
> why don't you free black people in US

404 Freedom not found, period.

**Comment by KOLANICH** [comment]
>Bull shit man, you think you know real PRC from those fake news that your government shows you?what a joke?

Not government, but a man I know, who has visited PRC on business. But he is OK with the things going on there, it's me who is not OK with them, that's why he has already visited, but I am not going to, and going to stay away from buying chinese electronics, especially featurephones/smartphones/tablets/phablets and routers, too.

**Comment by 23333333333333333333333333333333333333** [comment]
飞他🐎的，像个🤡，真可怜！

**Comment by Lin-Buo-Ren** [comment]
政治的定義是「眾人之事」，現在還在說技術歸技術政治歸政治的你一定是誤會了什麼

Stand with Hong Kong!  F__k the CCP!

**Comment by minews** [comment]
HongKong is part of China as always!

**Comment by KOLANICH** [comment]
People protest not against HK being a part of China (as an area on Earth surface). Of course HK as a spot on Earth surface cannot split of China in any observable time. It'd just require the technologies not available for a very long indefinite period of time.

People protest against treating them as shit, as PRC treats everyone in the world this way (including everyone taking part in this discussion), especially the people that have ever been on parts of mainland China under PRC rule (including PRC own "elite" slaves). And only an immoral person can approve and/or advocate treating people as shit.

**Comment by Lin-Buo-Ren** [comment]
In your dreams <3.

minews <notifications@github.com> 於 2020年12月13日 週日 下午2:17 寫道：

**Comment by bill4u** [comment]
无论你们怎么叫嚣，香港终究是中国的，你们能改变这个事实吗？除了在这可怜的叫嚣外，你们有什么实质的办法吗？没有。逞口舌之快，意淫一下有什么用？你的老巢，台湾也是。你们有办法把台湾岛搬到加勒比去吗？没有！台湾过去、现在以及将来都将在我们的东风快递配送范围内，不打你不是打不了你。跳梁小丑，还是先关心一下台湾岛内人员的生死吧，可怜香港？笑话。垃圾。

**Comment by bill4u** [comment]
> In your dreams <3. minews <notifications@github.com> 於 2020年12月13日 週日 下午2:17 寫道：
> […](#)
对你们这种人不用讲道理，打就对了。吃里扒外的杂种。无论你的腐国主子怎么跳怎么叫，有本事来香港驻军啊？~~

**Comment by vivalg** [comment]
傻逼你别用不就行了

**Comment by quarkdownload** [comment]
> 傻逼你别用不就行了
我日你妈了个臭逼
![77a98a08a8cdcb236ff2e8f79ff16ba5fe51f8bc2a9dc30e46ecda28cf7faf4e](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/35839307/2da6d08b-ae59-4c24-ba93-1ad55bd1bcf6)
